resources/lang/en/accounts.php

```diff
@@ -47,7 +47,7 @@
             'user' => [
                 'user_from' => 'current location',
                 'user_interests' => 'interest',
-                'user_msnm' => 'skype',
```

Comment:
are you going to go and change osu! to Osu! next?

in this context it is "skype", along with "twitter" and "interest". please do not just change these